spyder/app/mainwindow.py

```diff
@@ -223,6 +223,22 @@ def get_focus_python_shell():
         return widget.shell
 
 
+def get_environment_infostr():
+    """Find if running inside a conda and/or venv

```

Comment:
Technically, shouldn't be this something like "Get the name of the conda or venv environment if running inside one" as the current wording implies it returns a bool of whether or not it is running in a specific env?